config/routes.rb

```diff
@@ -288,6 +288,8 @@
         defaults: { feature: feature }
       )
     end
+
+    resources :coronavirus_chatbot, only: :create
```

Comment:
should the route be more like coronoavirus_chatbot_tokengen or something? given that this endpoint is mainly just returning the valid token as per my understanding.

Reply:
how about `/v0/coronoavirus_chatbot/tokens` so that we can keep it railsy?

Reply:
sure - I take it tokens is better than token re: rails conventions?

Reply:
typically resources are pluralized as to reflect the real world, depening on inflector conventions. TL;DR yes